src/current/v24.2/child-metrics.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Child Metrics
+summary: Learn about high-cardinality child metrics enabled by the cluster setting server.child_metrics.enabled.
+toc: true
+docs_area: reference.metrics
+---
+
+The [cluster setting `server.child_metrics.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-server-child-metrics-enabled) enables the exporting of child metrics, which are additional [Prometheus]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#prometheus-endpoint) time-series with extra labels.
+
+This page provides details on various metrics and their potential child metrics, which are determined by the specific feature utilized by the cluster. The number of these child metrics can significantly increase based on their associated labels, also known as "high cardinality". Refer to the information here to understand the potential size of the Prometheus scrape payload for your workload when child metrics are enabled.
+
+## Enable child metrics
+
+`server.child_metrics.enabled` is disabled by default. To enable it, use the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING server.child_metrics.enabled = true;
+~~~
+
+## All clusters
+
+An RPC (Remote Procedure Call) connection is a communication method used in distributed systems, like CockroachDB, to allow one program to request a service from a program located in another computer on a network without having to understand the network's details. In the context of CockroachDB, RPC connections are used for inter-node communication. For instance, if Node 1 sends a request to Node 2 and Node 2 dials back (sends request back to Node 1), it ensures that communication is healthy in both directions. This is referred to as a "bidirectionally connected" and "heartbeating" RPC connection.
```

Comment:
```suggestion
An RPC (Remote Procedure Call) connection is a communication method used in distributed systems, like CockroachDB, to allow one program to request a service from a program located in another computer on a network without having to understand the network's details. In the context of CockroachDB, RPC connections are used for inter-node communication. For instance, if Node 1 sends a request to Node 2, and Node 2 dials back (sends request back to Node 1), it ensures that communication is healthy in both directions. This is referred to as a "bidirectionally connected" and "heartbeating" RPC connection.
```
I think you need the comma there, other it sounds like you are sending a request to Node 2 AND Node 2.